.github/workflows/scripts/start-kind.sh

```diff
@@ -47,8 +47,10 @@ WorkerChartOptions:
     "operator.image": "worker-operator"
     "operator.tag": "${GITHUB_HEAD_COMMIT}"
 TestSuitesEnabled:
+  EmptySuite: true
   HubSuite: false
   WorkerSuite: true
+  IstioSuite: false
```

Comment:
please enable this before merging the PR

Reply:
I believe there is an issue with the istiosuite